index.html

```diff
@@ -33,6 +33,8 @@
 
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-128972978-1"></script>
+    <!--Press Start Font-->
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
```

Comment:
I think we should import multiple google fonts in one call, can append it [here](https://github.com/ckcollab/ckcollab/blob/gh-pages/assets/css/global.css#L1) I think?

probably something like...
```
@import url("//fonts.googleapis.com/css?family=Montserrat:400|Press+Start+2P:400");
```